core/store/Cargo.toml

```diff
@@ -14,7 +14,13 @@ bytesize = { version = "1.1", features = ["serde"] }
 derive_more = "0.99.3"
 elastic-array = "0.11"
 enum-map = "2.1.0"
-rocksdb = { version = "0.18.0", default-features = false, features = ["snappy", "lz4", "zstd", "zlib"] }
+rocksdb = { version = "0.18.0", default-features = false, features = [
```

Comment:
I think we have rocks in three different Cargo.tomls, each needs to be updated

Reply:
I’m waiting for Cargo 1.64 to fix that.

Reply:
Updated all cargo.tomls that mention rocksdb.